CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 
 ## [Unreleased]
 
+## [5.0.1]
+
+### Added
+- Support to WFS reader using the original data from download
+- Add an option to overwrite content in Geopackage Writer instead of appending content
+- Add a parameter to the GeoPackage writer that allows creating tables for all mapping-relevant target types
+- Add the creation of a .cpg file when a .shp file is beeing exported
```

Comment:
> file is beeing exported

Misspelled `being`? 

I think instead of just `.shp` it would be great to mention Shapefile.

As this line is not added, please also take care to merge the related PR before you proceed with the release builds.